docs/book/src/developer/architecture/controllers/machine-pool.md

```diff
@@ -96,6 +96,26 @@ The `status` object **may** define several fields that do not affect functionali
 * `failureReason` - is a string that explains why a fatal error has occurred, if possible.
 * `failureMessage` - is a string that holds the message contained by the error.
 
+Example:
+```yaml
+kind: MyMachinePool
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+spec:
+    providerIDList:
+      - cloud:////my-cloud-provider-id-0
+      - cloud:////my-cloud-provider-id-1
+status:
+    ready: true
+```
+
+#### Externally Managed Autoscaler
```

Comment:
Given it's a contract change/addition I would suggest to add this to the v1.2-v1-3 migration guide as well (v1.2-to-v1.3.md)

Reply:
made an initial attempt at describing this change in the migration guide